main.py

```diff
@@ -0,0 +1,6 @@
+# I added an entry point here.
+# If you don't want this, please remove it.
+
+from homr.main import main
+
+main()
```

Comment:
I think I once added the same thing and then removed it as  "python homr/main.py" (in a poetry shell) was convenient enough. I'm fine having this, I was just wondering for the reason.